applications/templates/application.html

```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
+{% load snippet %}
```

Comment:
nit: I like putting a blank line before `load`s so they're a group separate from the `extends`.